consensus/core/src/block_verifier.rs

```diff
@@ -237,6 +247,68 @@ impl BlockVerifier for SignedBlockVerifier {
         }
         Ok(())
     }
+
+    fn verify_misbehavior_reports(
```

Comment:
I'm missing the `target` verification and the ordering between `first` and `second` in the equivocation part